gke-deploy/core/container/container.go

```diff
@@ -17,17 +17,83 @@ import (
 	"fmt"
 
 	"github.com/GoogleCloudPlatform/cloud-builders/gke-deploy/services"
+	"github.com/google/go-containerregistry/pkg/name"
 )
 
 const (
 	imageDigestFormat = "value(image_summary.digest)"
 )
 
-// GetDigest uses an image string to get its corresponding digest.
-func GetDigest(ctx context.Context, image string, gs services.GcloudService) (string, error) {
-	digest, err := gs.ContainerImagesDescribe(ctx, image, imageDigestFormat)
+// Image is an alias to name.Reference.
+type Image = name.Reference
```

Comment:
Is this alias useful? Can you just pass around `name.Reference`s wherever needed instead?

(Technically, "container" is a misnomer here anyway, you're dealing with "images", not any running "containers")

Reply:
Removed the reference.